cmd/main.go

```diff
@@ -30,11 +30,6 @@ func main() {
 
 	flag.Parse()
 
-	if len(os.Args) == 1 || remoteSki == nil || *remoteSki == "" {
```

Comment:
Without providing an SKI the service can not run meaningful right now. One would have to also integrate accepting any (because there is no UX to deny them) incoming remote pairing requests.

Otherwise block should stay.

Reply:
the current implementation isn't able to generate the cert and print the SKI for pairing.
The idea behind that is providing an easier startup for pairing the cemd with a other eebus device.
Therefore the cemd ski is needed if the target is not auto accept the ski.

Reply:
Then I suggest simply moving this code blog before `configuration, err := eebusapi.NewConfiguration`